src/Utils.cc

```diff
@@ -89,6 +89,72 @@ float screenScalingFactor()
   return ratio;
 }
 
+/////////////////////////////////////////////////
+void screenScalingFactor(float &_xScale, float &_yScale)
+{
+  // todo(anyone) set device pixel ratio for high dpi displays on Windows
+#ifdef __linux__
```

Comment:
Should we set the ratio to 1.0 for non-linux like the other `screenScalingFactor` function does?